doc/en/reference.rst

```diff
@@ -971,12 +968,6 @@ test functions and methods. Can be either a single mark or a list of marks.
 
     pytestmark = [pytest.mark.integration, pytest.mark.slow]
 
-PYTEST_DONT_REWRITE (module docstring)
```

Comment:
This is not really a global variable, and is already mentioned in detail in the `assertion` chapter, so decided to remove this from here as it is pretty niche.